velox/exec/StreamingAggregation.cpp

```diff
@@ -215,6 +229,15 @@ void StreamingAggregation::assignGroups() {
       }
     }
   }
+
+  groupBoundaries_.clear();
+  for (vector_size_t i = 1; i < numInput; ++i) {
+    if (inputGroups_[i] != inputGroups_[i - 1]) {
+      groupBoundaries_.push_back(i);
```

Comment:
@Yuhta My reading of this code suggests that this comment is incorrect. It looks like 'groupBoundaries' contains index of the first row in the new group, not index of the last row in the 'current' group. Am I missing something?

> `groupBoundaries` is the indices into `groups` indicating the last row of each group.

Reply:
Yes you are correct, let me correct the comment